wgsl/index.bs

```diff
@@ -684,29 +684,29 @@ may be read, without the use of a sampler.
 See [[#texture-builtin-functions]].
 
 <pre class='def'>
-`texture_ro_1d<image_storage_type>`
+`texture_storage_ro_1d<image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 1D 0 0 0 2 image_storage_type
 
-`texture_ro_1d_array<image_storage_type>`
+`texture_storage_ro_1d_array<image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 1D 0 1 0 2 image_storage_type
 
-`texture_ro_2d<image_storage_type>`
+`texture_storage_ro_2d<image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 0 0 0 2 image_storage_type
 
-`texture_ro_2d_array<image_storage_type>`
+`texture_storage_ro_2d_array<image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 0 1 0 2 image_storage_type
 
-`texture_ro_3d<image_storage_type>`
+`texture_storage_ro_3d<image_storage_type>`
   %1 = OpTypeImage type(image_storage_type) 3D 0 0 0 2 image_storage_type
 </pre>
 * type(image_storage_type) is one of `f32`, `i32`, or `u32`, depending on the storage type.
 
 When mapping to SPIR-V, a read-only storage texture variable must also have a `NonWritable` decoration.
 For example:
 
-<div class='example' heading='Mapping a texture_ro_1d variable to SPIR-V'>
+<div class='example' heading='Mapping a texture_storage_ro_1d variable to SPIR-V'>
   <xmp>
-      var<uniform_constant> tbuf : texture_ro_1d<rgba8unorm>;
+      var<uniform_constant> tbuf : texture_storage_ro_1d<rgba8unorm>;
```

Comment:
The storage class designation is no longer used.    Suggest we fix that in a future CL